content/en/serverless/installation/dotnet.md

```diff
@@ -32,6 +32,63 @@ Instrumenting .NET applications with the Datadog Lambda Extension is in beta.
 Datadog offers many different ways to enable instrumentation for your serverless applications. Choose a method below that best suits your needs.
 
 {{< tabs >}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda functions' configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+You can also add the command to your CI/CD pipelines to enable instrumentation for all your serverless applications. Run the command _after_ your normal serverless application deployment, so that changes made by the Datadog CLI command are not overridden.
```

Comment:
What is the command?

Reply:
The commands are down a bit under their respective headers. This is the same format used for the other runtimes as well and it just walks through what the CI is, how to set it up and then the commands to use.

Reply:
Okay, thanks. How about "instrumentation command"? Or "instrumentation and installation commands" if it's both? And linking it to the respective section(s)?